dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/imports/TrackerImportControllerTest.java

```diff
@@ -61,6 +62,22 @@ void shouldSucceedWhenAllValidParametersArePassed() {
             .content(HttpStatus.OK));
   }
 
+  @Test
+  void shouldReturnBadRequestWhenThereIsAnInvalidUidInThePayload() {
+    assertWebMessage(
+        "Bad Request",
+        400,
+        "ERROR",
+        "JSON parse error: Cannot construct instance of `org.hisp.dhis.common.UID`, problem: UID must be an alphanumeric string of 11 characters starting with a letter.",
+        POST(
+                "/tracker?async=false&reportMode=FULL"
+                    + "&validationMode=SKIP"
+                    + "&atomicMode=OBJECT"
+                    + "&skipRuleEngine=true",
+                Path.of("tracker/import_invalid_uid_payload.json"))
```

Comment:
Now that multi-line strings are supported favor that. This reduces the scope of the test setup. We have so many files 😅 Hard to know what's not used anymore, needed by what, ...